projects/playground/src/app/app.component.html

```diff
@@ -47,3 +47,8 @@ <h2>Artifacts</h2>
     </li>
   </ul>
 </div>
+
+<h2>Test Auth Guard</h2>
+<a routerLink="protected">Protected Route</a> |
```

Comment:
would it be better if we use an absolute path here?

```suggestion
<a routerLink="/protected">Protected Route</a> |
```

Reply:
Sure, also fine.